src/app/wizard/components/input/input.component.html

```diff
@@ -14,6 +14,16 @@
       </div>
       <span *ngIf="input.note"><span class="fa fa-info-circle"></span> <i>{{input.note}}</i></span>
     </span>
+    <!-- todo this might be a bit specific -->
+    <ul class="dropdown-menu" *ngIf="input.typeAheadData && input.hasFocus" [style.display]="input.clicked ? 'none' : 'block'" style="left: auto">
```

Comment:
Why ngIf tests if input.hasFocus?

Reply:
Only show the suggest box if the field has focus. Otherwise if there are multiple fields with suggest boxes there would be boxes shown all over the form